doc/nrf/ug_thread_tools.rst

```diff
@@ -34,25 +34,236 @@ See `nRF Thread Topology Monitor`_ for documentation.
+
+To program the nRF device with the RCP application, complete the following steps:
+
+#. Clone the OpenThread nRF528xx platform repository into the current directory:
+
+   .. code-block:: console
+
+      git clone --recursive https://github.com/openthread/ot-nrf528xx.git
+
+#. Enter the :file:`ot-nrf528xx` directory:
+
+   .. code-block:: console
+
+      cd ot-nrf528xx
+
+#. Install the OpenThread dependencies:
+
+   .. code-block:: console
+
+      ./script/bootstrap
+
+#. Build the RCP example for the hardware platform and the transport of your choice:
+
+   .. tabs::
+
+      .. tab:: nRF52840 Dongle (USB transport)
+
+         .. code-block:: console
+
+            rm -rf build
+            script/build nrf52840 USB_trans -DOT_BOOTLOADER=USB -DOT_THREAD_VERSION=1.2
+
+      .. tab:: nRF52840 Development Kit (UART transport)
+
+         .. code-block:: console
+
+            rm -rf build
+            script/build nrf52840 UART_trans -DOT_THREAD_VERSION=1.2
+
+   ..
+
+   This creates an RCP image at :file:`build/bin/ot-rcp`.
+#. Convert the RCP image to hexadecimal format:
+
+   .. code-block:: console
+
+      arm-none-eabi-objcopy -O ihex build/bin/ot-rcp build/bin/ot-rcp.hex
+
+#. Depending on the hardware platform, complete the following steps:
+
+   .. tabs::
+
+      .. tab:: nRF52840 Dongle (USB transport)
+
+         a. Install nRF Util:
+
+            .. code-block:: console
+
+               python3 -m pip install -U nrfutil
+
+            .. note::
+
+               The nRF Util version distributed officially through PyPI is not supported on Raspberry Pi.
+               To install a compatible version, execute the following commands:
+
+               .. code-block:: console
+
+                  sudo apt-get -y install libusb-1.0-0-dev sed
+                  pip3 install click crcmod ecdsa intelhex libusb1 piccata protobuf pyserial pyyaml tqdm pc_ble_driver_py pyspinel
+                  pip3 install -U --no-dependencies nrfutil==6.0.1
+                  export PATH="$HOME/.local/bin:$PATH"
+
+         #. Generate the RCP firmware package:
+
+            .. code-block:: console
+
+               nrfutil pkg generate --hw-version 52 --sd-req=0x00 \
+                --application build/bin/ot-rcp.hex --application-version 1 build/bin/ot-rcp.zip
+
+         #. Connect the nRF52840 Dongle to the USB port.
+         #. Press the **RESET** button on the dongle to put it into the DFU mode.
+            The LED on the dongle starts blinking red.
+         #. Install the RCP firmware package onto the dongle by running the following command, with ``/dev/ttyACM0`` replaced with the device node name of your nRF52840 Dongle:
+
+            .. code-block:: console
+
+               nrfutil dfu usb-serial -pkg build/bin/ot-rcp.zip -p /dev/ttyACM0
+
+      .. tab:: nRF52840 Development Kit (UART transport)
+
+         a. Program the image using the nrfjprog utility:
```

Comment:
Can we assume that nrfjprog is installed?

Reply:
I changed the sentence to be aligned with other usage of nrfutil in our documentation:

```
         a. Program the image using the nrfjprog utility (which is part of the `nRF Command Line Tools`_):
```